backend/krk_meetings/solver/ConnectionSolver.py

```diff
@@ -1,3 +1,4 @@
+import sys
```

Comment:
what's that for?

Reply:
it's redundant. I'll remove it and than merge